lib/plugins/aws/lib/monitorStack.js

```diff
@@ -107,21 +108,17 @@ module.exports = {
                   }
                 }
 
-                const operationPhrase = (() => {
-                  if (action === 'create') return 'Creating';
-                  if (action === 'update') return 'Updating';
-                  if (action === 'delete') return 'Deleting';
-                  return '';
-                })();
-
-                if (operationPhrase && (completedResources.size || inProgressResources.size)) {
-                  progress
-                    .get('main')
-                    .notice(
-                      `${operationPhrase} CloudFormation stack (${completedResources.size}/${
-                        completedResources.size + inProgressResources.size
-                      })`
-                    );
+                if ((completedResources.size || inProgressResources.size) && action !== 'create') {
+                  const progressMessagePrefix = (() => {
+                    if (action === 'update') return 'Updating';
+                    if (action === 'delete') return 'Deleting';
```

Comment:
It should result in `Removing` instead of `Deleting` to be consistent with our wording (it has been changed in one of recent PRs)